package.json

```diff
@@ -15,7 +15,7 @@
     "@graasp/chatbox": "2.0.0",
     "@graasp/query-client": "1.1.3",
     "@graasp/sdk": "1.1.1",
-    "@graasp/translations": "1.15.0",
+    "@graasp/translations": "github:graasp/graasp-translations#180-published-tab",
     "@graasp/ui": "3.2.1",
```

Comment:
If you rebase these should be updated to the latest versions. If you encounter issues with types make sure to update `immutable` to 4.3.1